Measures/AddingInternalLoads/measure.rb

```diff
@@ -43,6 +43,21 @@ def arguments(model)
 	args << OpenStudio::Measure::OSArgument.makeStringArgument("PeopleActivityScheduleSonntag", true)
```

Comment:
Sonntag => Sunday

Reply:
Is adressed in a later commit (branch rename-measure-arguments)